### ISSUE TITLE
Fermata mirror fs 

### ISSUE THREAD

**Issue by Planetiptv** [AndreyPavlenko/Fermata#568]
@AndreyPavlenko can u help me?
I have uconnect 7"
Fermata auto work in full screen
Fermata mirror fs not full screen.what i have to do?
![IMG_20241222_130436_060801](https://github.com/user-attachments/assets/87bb3a4b-39d9-409c-b747-551ed6df06c6)
![IMG_20241222_130455_060835](https://github.com/user-attachments/assets/7ecad824-a4b8-4b36-a527-30ad10c1e06c)

**Comment by AndreyPavlenko** [comment]
This is indeed full screen. Unfortunately, you can't get rid of AA buttons. At least without root.
If you mean the black gaps on top and bottom, this is because the phone screen size is not proportional to the car screen size.

**Comment by Planetiptv** [comment]
I understand. Thank u

**Comment by AndreyPavlenko** [comment]
I don't think you can change the screen aspect ratio without root.

**Comment by Planetiptv** [comment]
Ok . Maybe can i put aa button from leftside to down
![IMG_20241222_130436_060801~2](https://github.com/user-attachments/assets/cf4f614e-a16e-4c09-89ff-6d6f114e586a)

**Comment by AndreyPavlenko** [comment]
In AA settings there is a "Change layout" option, but I'm not sure if it's what you need. The AA layout depends on the screen size and resolution. If you have AAWireless, you can change DPI.

**Comment by Planetiptv** [comment]
@AndreyPavlenko after many try i concluse that i cant change dpi without software on my unroot phone. So i decise to buy aawireless2.

**Comment by shen9175** [comment]
> This is indeed full screen. Unfortunately, you can't get rid of AA buttons. At least without root. If you mean the black gaps on top and bottom, this is because the phone screen size is not proportional to the car screen size.

Hi, I just bought my new phone (6.8 inch) and installed the latest 1.9.9 version. It seems the fullscreen is smaller than my previous phone (I mean there are the black_gas on the top and bottom) My car is using 12-inch screen and last year the same car my old phone screen is smaller than my new phone around 5.8 inch. But I compared the video I recorded with my old phone connect with Fermata auto in the same car, it indeed larger than the fullscreen size (there is no top and bottom black gaps)  that I got with my new phone. Maybe my old phone using 1.9.8 (I forgot, I lost it). Is there a way to let fullscreen bigger and remove the top and bottom gaps?


And my phone is rooted with LSPosed, is there a way to get rid of AA buttons on the leftside? how?

**Comment by AndreyPavlenko** [comment]
The gaps are because the phone screen dimensions are not proportional to the car screen dimensions.

**Comment by shen9175** [comment]
> The gaps are because the phone screen dimensions are not proportional to the car screen dimensions.
@AndreyPavlenko 

Yeah, today I checked it very carefully, I found the new phone is actually longer than my old phone. And after projected to car screen the two vertical sides already hit the car's screen's edge. When apps like youtube play video, it only use center part of the whole screen, leaves the two sides black and one is for camera and the other side is for virtual android 3-keys (back, home and app-list). I found this because youtube put the video's authur profile picture on the left black gap, then I realized that the two vertical black gaps are not the real black gaps, it's can show something but app leaves to black.  This is why two vertical sides are black, the screen is actually used but the app does not want to show video there.

If in the future Fermata Auto can have an option to show 16:9 or whatever the user want (depend on car screen), Fermata Auto can  show only the center of screen to stretch to 16:9  (or whatever the user specify or the common aspect ratios).

For example, my phone is 3168x1440 (19.2:9), if we project this phone 19.2:9 screen to car16:9 screen, 3168 should hits two edges to cause the 1440 part has two black gaps. But the app will show 16:9 video only in the center of 3168, which means the full screen video is 2560x1440 (16:9), then two vertical black gaps width is (3168-2560)/2=304, then Fermata auto can just project (304,0) to (2560+304=2864, 1440)  which is the rectangle [304,0 => 2864, 1440] as the Fullscreen to project to car screen.  This method just depend on target aspect ratio and the phone's physical resolution. If can have an option to let user to choose, it will be great.

**Comment by AndreyPavlenko** [comment]
I don't know if it's possible to change the phone screen resolution without root.

**Comment by shen9175** [comment]
I'm root user.

**Comment by AndreyPavlenko** [comment]
You may use the app without root, but for A14 you need AAWireless.
If you have root, try this (the first link from google) - https://www.youtube.com/watch?v=-a9OpbuGcO8

**Comment by shen9175** [comment]
> You may use the app without root, but for A14 you need AAWireless. If you have root, try this (the first link from google) - https://www.youtube.com/watch?v=-a9OpbuGcO8

yeah, this Easy DPI changer really works! Just change the screen resolution from 3168x1440 to 2560x1440, the car screen projection become true full screen, no or very little black gaps. Thanks!

**Comment by nochlab1** [comment]
https://play.google.com/store/apps/details?id=com.farmerbb.secondscreen.free

This app can change dpi without root

**Comment by Gnomesu** [comment]
Tell me how to remove the stripes on the left and right on the head unit screen. The smartphone is rooted, what applications can I use for this? (Подскажите как убрать полосы слева и справа на экране головного устройства. Смартфон рутирован, какие приложения могу для этого использовать?)
